website/docs/workloads/specview/specview.md

```diff
@@ -0,0 +1,79 @@
+# SPECviewperf Benchmark
+The SPECviewperf® 2020 v3.0 benchmark, released on December 9, 2021, is the worldwide standard for measuring graphics performance based on professional applications.
+The benchmark’s workloads, called viewsets, represent graphics content and behavior extracted from professional applications, without the need to install the applications themselves. 
+
+Applications represented by viewsets in SPECviewperf 2020 include Autodesk 3ds Max and Maya for media and entertainment; Dassault Systèmes Catia and Solidworks, PTC Creo, and Siemens NX for CAD/CAM; and two datasets representing professional energy and medical applications.
+
+* [SPECviewperf 2020 v3.0](https://gwpg.spec.org/benchmarks/benchmark/specviewperf-2020-v3-0/)
+
+## What is Being Measured 
+The benchmark measures the 3D graphics performance of systems running under the OpenGL and DirectX application programming interfaces.
+
+## System Requirements
+* Microsoft Windows 10 Version 1709 (Fall Creators Update / RS3) or newer
```

Comment:
As I mentioned elsewhere, if we need psexec for windows graph API, we need to mention here